broadinstitute/orsp-pub
# BTRX 614 Review Category Report & Search Autocomplete bug

## Addresses
https://broadinstitute.atlassian.net/browse/BTRX-614
## Changes
- Add new component for Review Category Report
- Include credentials in fetch to get access to the specified endpoints
## Testing
- As a logged user, go to a Admin menu, Review Category Report
- Go to Search page and filter by ORSP Identification, autocomplete should work
- Go to Search page and filter by Person, autocomplete should work
---
